plexapi/client.py

```diff
@@ -540,20 +542,65 @@ def setStreams(self, audioStreamID=None, subtitleStreamID=None, videoStreamID=No
 
     # -------------------
     # Timeline Commands
-    def timeline(self, wait=1):
-        """ Poll the current timeline and return the XML response. """
-        return self.sendCommand('timeline/poll', wait=wait)
+    def timelines(self, wait=0):
+        """Poll the client's timelines, create, and return timeline objects.
+           Some clients may not always respond to timeline requests, believe this
+           to be a Plex bug.
+        """
+        t = time.time()
+        if t - self._timeline_cache_timestamp > 1:
+            self._timeline_cache_timestamp = t
+            timelines = self.sendCommand(ClientTimeline.key, wait=wait) or []
+            self._timeline_cache = [ClientTimeline(self, data) for data in timelines]
+
+        return self._timeline_cache
 
-    def isPlayingMedia(self, includePaused=False):
-        """ Returns True if any media is currently playing.
+    @property
+    def timeline(self):
+        """Returns the active timeline object."""
+        return next((x for x in self.timelines() if x.state != 'stopped'), None)
+
+    def isPlayingMedia(self, includePaused=True):
+        """Returns True if any media is currently playing.
 
             Parameters:
                 includePaused (bool): Set True to treat currently paused items
-                    as playing (optional; default True).
+                                      as playing (optional; default True).
         """
-        for mediatype in self.timeline(wait=0):
-            if mediatype.get('state') == 'playing':
-                return True
-            if includePaused and mediatype.get('state') == 'paused':
-                return True
-        return False
+        state = getattr(self.timeline, "state", None)
+        return bool(state == 'playing' or (includePaused and state == 'paused'))
+
+
+class ClientTimeline(PlexObject):
+    """Get the timeline's attributes."""
+
+    key = 'timeline/poll'
+
+    def _loadData(self, data):
+        self._data = data
+        self.address = data.attrib.get('address')
+        self.autoPlay = utils.cast(int, data.attrib.get('autoPlay'))
```

Comment:
This is delivered as an `int` but it represents a `bool`. Should we change the casting?